diff -urN pyperl-1.0.org/svrv_object.c pyperl-1.0/svrv_object.c
--- pyperl-1.0.org/svrv_object.c	2001-03-01 23:36:16.000000000 +0100
+++ pyperl-1.0/svrv_object.c	2006-05-22 11:15:18.000000000 +0200
@@ -7,6 +7,9 @@
 #include <perl.h>
 #include <Python.h>
 
+#define PERL_CORE
+#include "embed.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
@@ -18,6 +21,7 @@
 #include "lang_map.h"
 #include "try_perlapi.h"
 
+
 #ifdef MULTI_PERL
 static int
 owned_by(PySVRV *self, refcounted_perl *my_perl)
diff -urN pyperl-1.0.org/try_perlapi.pl pyperl-1.0/try_perlapi.pl
--- pyperl-1.0.org/try_perlapi.pl	2001-01-10 19:13:34.000000000 +0100
+++ pyperl-1.0/try_perlapi.pl	2006-05-22 11:15:46.000000000 +0200
@@ -20,6 +20,9 @@
 #include <perl.h>
 #include <Python.h>
 
+#define PERL_CORE
+#include "embed.h"
+
 #include "try_perlapi.h"
 #include "perlmodule.h"
 #include "lang_lock.h"
